modules/core/src/main/scala/com.snowplowanalytics.snowplow.bigquery/processing/LegacyColumns.scala

```diff
@@ -125,8 +125,8 @@ object LegacyColumns {
               case TabledEntity.Context       => LegacyMode.Repeated
             }
             val columnName  = legacyColumnName(entityType, schemaKey)
-            val legacyField = LegacyField.build(columnName, schema, false).setMode(mode).normalized
-            val v2Field     = legacyFieldToV2Field(legacyField).copy(accessors = Set.empty)
+            val legacyField = LegacyField.build(columnName, schema, false).setMode(mode)
+            val v2Field     = V2Field.normalize(legacyFieldToV2Field(legacyField)).copy(accessors = Set.empty)
```

Comment:
What is the reason of this change?

Now we normalize legacy fields using V2 normalization. Just want to be sure it's 100% safe 😁

Reply:
If we normalize the v1 field, then we drop information about whether the original field name was snake case or camel case.  So if the original field name was camel case then the casting fails [on this line](https://github.com/snowplow/schema-ddl/blob/414a2d76902b8653433154a1c7029b15768ed888/modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/parquet/Caster.scala#L176).

I discovered this when I tried to load some events with the yauaa context.

Hmmm, maybe I should change [these test schemas](https://github.com/snowplow-incubator/snowplow-bigquery-loader/tree/v2/modules/core/src/test/resources/iglu-client-embedded/schemas/myvendor/myschema/jsonschema) to include some camel case fields.  And then this PR would be fixing a failing a test.

Reply:
Ok, that makes sense!

Last question - why do we need `.copy(accessors = Set.empty)` here?

Reply:
It doesn't make any difference to how the loader works.  But it makes it easier to write tests.

If I remove the `.copy(...)`, then I would also have to change [this line in the Spec](https://github.com/snowplow-incubator/snowplow-bigquery-loader/blob/819f267ed3795adef0232e5de9e7727a8aaa8b54/modules/core/src/test/scala/com.snowplowanalytics.snowplow.bigquery/processing/LegacyColumnsResolveSpec.scala#L65) to be like...

```scala
val expectedField = Field("unstruct_event_myvendor_myschema_7_0_0", expectedStruct, Nullable, "unstruct_event_myvendor_myschema_7_0_0")
```

(+ lots of other places)

...which is fine, but it's a bit annoying to repeat the field name each time in the expected field.

Reply:
Actually ignore that... it's completely wrong.  I'll try removing the `.copy(....)`, because it might work.